### ISSUE TITLE
May I ask how accurate your initial pose got from integral regression is?

### ISSUE THREAD

**Issue by ArchNew** [lingtengqiu/OPEC-Net#9]
According to the paper "Integral human pose regression", it seems that it needs too many epochs to directly train a pose estimation net with integral regression as its loss. And I've tried with HRNet and SimpleBaselines, I never got anywhere near the accuracy based on heatmaps.

I also experimented on directly computing keypoints coordinates from heatmaps using the integral regression function. The accuracy is too low to be useful.

I'm curious how do you get your initial pose from the integral regression. Just get a low-accuracy one computed from heatmaps via integral regression function, or you fine-tuned your model using integral regression?

**Comment by lingtengqiu** [comment]
Hi, in our experiments, we find that using intergral function to estimate human poses whose accuracy actually is less than estimation based on heatmap. The answer why we replace heatmap into intergral function or linear function is that making the framework  differentiable.

**Comment by ArchNew** [comment]
Thanks for your quick response.
